.github/actions/app-run-local-env/serve-frontend.py

```
@@ -0,0 +1,18 @@
+
+class CorsRequestHandler(SimpleHTTPRequestHandler):
+    def end_headers(self):
+        self.send_header("Access-Control-Allow-Origin", "*")
+        super().end_headers()
+
+
+if len(sys.argv) != 2:
+    raise SystemExit("usage: serve-frontend.py DIST_DIRECTORY")
+
+handler = partial(CorsRequestHandler, directory=sys.argv[1])
+ThreadingHTTPServer(("0.0.0.0", 8080), handler).serve_forever()
```

Ruff (0.16.1)

[error] Possible binding to all interfaces

(S104)